doc-content/enterprise-only/integration/dev-studio-import-local-git-repo-proc.adoc

```diff
@@ -1,12 +1,12 @@
 [id='dev-studio-import-local-git-repo-proc']
 = Importing a local Git repository
-You can import a local git repository to use with Red Hat JBoss Developer Studio.
+You can import a local Git repository to use with Red Hat Developer Studio.
```

Comment:
Again, you can do this later, but I would like to know why I would want to use Git with Dev Studio.

Reply:
Is it permissible to duplicate or expand upon the chapter's introductory paragraph?

Reply:
Keeping open to revisit